sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1503 @@
+    "mlflow_client = MlflowClient()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "#### Get the AutoML parent Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412506460
+    },
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "job_name = returned_job.name\n",
+    "\n",
+    "# Example if providing an specific Job name/ID\n",
+    "# job_name = \"591640e8-0f88-49c5-adaa-39b9b9d75531\"\n",
+    "\n",
+    "# Get the parent run\n",
+    "mlflow_parent_run = mlflow_client.get_run(job_name)\n",
+    "\n",
+    "print(\"Parent Run: \")\n",
+    "print(mlflow_parent_run)\n",
+    "\n",
+    "# Print parent run tags. 'automl_best_child_run_id' tag should be there.\n",
+    "print(mlflow_parent_run.data.tags)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "#### Get the AutoML best child run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670412507109
+    },
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Get the best model's child run\n",
+    "best_child_run_id = mlflow_parent_run.data.tags[\"automl_best_child_run_id\"]\n",
+    "print(\"Found best child run id: \", best_child_run_id)\n",
+    "\n",
+    "best_run = mlflow_client.get_run(best_child_run_id)\n",
+    "\n",
+    "print(\"Best child run: \")\n",
+    "print(best_run)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Model Evaluation and Forecasting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "### 6.1 Download the best model\n",
+    "Access the results (such as models, artifacts, metrics) of a previously completed AutoML Run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Create local folder\n",
+    "local_dir = \"./artifact_downloads\"\n",
+    "if not os.path.exists(local_dir):\n",
+    "    os.mkdir(local_dir)\n",
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
```

Comment:
Let us use code
```python
mlflow.artifacts.download_artifacts(run_id=mlflow_best_run.info.run_id, artifact_path='outputs', dst_path=local_dir)
```
To avoid warning.

Reply:
Done. Thanks!